packages/core-components/src/components/ProgressBars/Gauge.tsx

```diff
@@ -52,21 +52,33 @@ type Props = {
   inverse?: boolean;
   unit?: string;
   max?: number;
+  getColor?: GetColor;
 };
 
+type GetColorArgs = {
+  palette: BackstageTheme['palette'];
+  value: number;
+  inverse?: boolean;
+  max?: number;
+};
+
+export type GetColor = (
+  args: GetColorArgs,
+) => string | BackstageTheme['palette']['error'];
```

Comment:
Return type here should just be a `string` right?

Reply:
Although looking at it now maybe I've just gotten hopelessly confused here and it should just return a string.